client/package.json

```diff
@@ -26,12 +26,14 @@
     "@babel/preset-env": "^7.21.4",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.4",
+    "@jridgewell/sourcemap-codec": "^1.4.15",
+    "@rollup/plugin-terser": "^0.4.3",
```

Comment:
Kindly indicate in the comments why you used these two libraries.

Reply:
Installed those libraries while trying to work on PWA and forgot to remove them while pushing this code on splash screens